certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+                  "x-ms-summary": "Error message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "List contract's documents",
+        "description": "Retrieve all documents bound to a contract",
+        "operationId": "ListContractDocuments",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Contract ID",
+            "description": "ID of the contract to retrieve documents",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      },
+      "post": {
+        "summary": "Upload a document and attach it to a contract",
+        "description": "Upload a document to the environment and attach it to a specific contract",
+        "operationId": "UploadDocumentAndAttachToContract",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Contract ID",
+            "description": "ID of the contract to attach a new document to",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "required": true,
+            "default": "multipart/form-data; boundary=abcde"
+          },
+          {
+            "name": "file",
+            "in": "formData",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "binary"
+            },
+            "x-ms-summary": "Files",
+            "description": "Files binaries to upload"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "documents parts added",
+            "schema": {
+              "$ref": "#/definitions/DocumentAdded"
+            }
+          }
+        }
+      }
+    },
+    "/contracts/{id}/documentresult/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "A PDF File",
+            "schema": {
+              "type": "file",
+              "title": "PDF Document Binary"
+            }
+          }
+        },
+        "summary": "Download the contract's documents",
+        "produces": [
+          "application/pdf"
+        ],
+        "description": "Retrieve a PDF file that contains all the documents from a contrat",
```

Comment:
@QwerioTEAM all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations), and all file should be free of grammatical errors or typos. 
here you have "contrat" please fix to "contract"